Documentation/security/policy/language.rst

```diff
@@ -39,14 +39,14 @@ can talk to each other. Layer 3 policies can be specified using the following me
   above. DNS information is acquired by routing DNS traffic via a proxy.
   DNS TTLs are respected.
 
-.. _Labels based:
+.. _Endpoints based:
 
-Labels Based
-------------
+Endpoints Based
+---------------
 
-Label-based L3 policy is used to establish policy between endpoints inside the
-cluster managed by Cilium. Label-based L3 policies are defined by using an
-`EndpointSelector` inside a rule to choose what kind of traffic that can be
+Endpoints-based L3 policy is used to establish policy between endpoints inside
+the cluster managed by Cilium. Endpoints-based L3 policies are defined by using
+an `EndpointSelector` inside a rule to choose what kind of traffic that can be
```

Comment:
"choose" is not as specific as "select". "that" is extraneous.
```suggestion
an `EndpointSelector` inside a rule to select what kind of traffic can be
```

Reply:
Applied these changes, thanks for the suggestions.